### PR TITLE
Add new recipe for granola

### PR DESCRIPTION
**Pull request by nclz**
#57 
Here is a recipe for granola.

### CODE REVIEW

File: recipes/granola.qmd [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "Granola"
+image: "Add the URL to the image here"
```
Comment (lnnrtwttkhn) [inline recipes/granola.qmd:3]: Please add an image using a weblink.

File: recipes/granola.qmd [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+image-alt: "Add the URL to the image here"
```
Comment (lnnrtwttkhn) [inline recipes/granola.qmd:4]: Please add the same weblink for the image.

File: recipes/granola.qmd [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+categories:
+  - Add a list of catgeories here
```
Comment (lnnrtwttkhn) [inline recipes/granola.qmd:6]: Please add a few category labels, otherwise delete.

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Thank for the PR @nclz! I reviewed your PR and have requested additional changes. It would be great, if yo could take care of them. Thank you!

File: recipes/granola.qmd (hunk 1 above)
Comment (nclz) [reply recipes/granola.qmd]: Done! But not sure if it works?

File: recipes/granola.qmd (hunk 2 above)
Comment (nclz) [reply recipes/granola.qmd]: Done - but same question as above.

**Comment by nclz** [comment]
Thank you @lnnrtwttkhn for reviewing and your comments. I have implemented your requested changes. Please let me know, if you want me to make any additional changes.

File: recipes/granola.qmd [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+title: "Granola"
+image: https://upload.wikimedia.org/wikipedia/commons/c/ca/Banana_Walnut_Granola_%286796918872%29.jpg 
+image-alt: https://commons.wikimedia.org/wiki/File:Banana_Walnut_Granola_(6796918872).jpg
+categories:
+  - sweet
+  - vegetarian
+---
```
Comment (lnnrtwttkhn) [inline recipes/granola.qmd:8]: Please add your author information to the YAML header like this:

```YAML
author: <firstname> <lastname>
```

File: recipes/granola.qmd [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Preheat the oven.
+1. Chop the nuts, if desired.
+1. Mix all the ingredients (except the dried fruits, cacao nibs and chia seeds).
+1. Spread it on a baking sheet.
+1. Put it into the oven. Bake until the granola is golden brown.
+1. Let the granola cool. Add the dried fruits and chia seeds.
+1. Use it as a topping or enjoy it for breakfast. 
+
+
```
Comment (lnnrtwttkhn) [inline recipes/granola.qmd:34]: Please remove unnecessary newlines. Thanks!

**Review by lnnrtwttkhn — CHANGES_REQUESTED** [review]
Thanks for the additional changes @nclz! I've added a few last minor comments for additional changes. Would be lovely if you could still add them! Thanks!

**Comment by nclz** [comment]
Thank you for your feedback. I have implemented your requested changes.

**Comment by lnnrtwttkhn** [comment]
Perfect, thank you @nclz! Ready to merge :rocket:

**Comment by lnnrtwttkhn** [comment]
@allcontributors please add @nclz for bug, content, code

**Pull request merged**